src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10107,7 +10107,8 @@ void Player_Init(Actor* thisx, PlayState* play2) {
     Player_SetEquipmentData(play, this);
     this->prevBoots = this->currentBoots;
     Player_InitCommon(this, play, gPlayerSkelHeaders[((void)0, gSaveContext.save.linkAge)]);
-    this->giObjectSegment = (void*)(((uintptr_t)ZELDA_ARENA_MALLOC(0x3008, "../z_player.c", 17175) + 8) & ~0xF);
+    this->giObjectSegment =
+        (void*)(((uintptr_t)ZELDA_ARENA_MALLOC(0x1000 * LANGUAGE_MAX + 8, "../z_player.c", 17175) + 8) & ~0xF);
```

Comment:
This is odd, why would the amount of languages matter for the gi object segment?
Im see the math works out this way so it matches, but conceptually it doesnt make sense to me

Reply:
The size of the title cards has changed since there's 2 languages instead of 3, and each language is 0x1000 image bytes. And I think the gi objects still fit because there's an assert in func_8083AE40 that the size is less than 0x2000

Reply:
right yeah, i just forgot about the shared use. might be worth a comment just so people can avoid the confusion i had, but not sure

Reply:
Good idea, added a comment